Description: Set info filename
 Set filename of generated texi file, and adapt Makefile to expect the
 correct output filename.
Author: Andreas Metzler <ametzler@debian.org>
Origin: vendor
Forwarded: no
Last-Update: 2013-09-29

--- a/doc-docbook/Makefile
+++ b/doc-docbook/Makefile
@@ -111,6 +111,7 @@ filter.txt:   filter-txt.xml Tidytxt MyS
 
 filter.info:  filter-info.xml
 	      $(DOCBOOK2TEXI) filter-info.xml
+	      if test -e filter.texi ; then mv filter.texi exim_filtering.texi ; fi
 	      perl -ne 's/conceptindex/cindex/;s/optionindex/findex/;print;' \
 		<exim_filtering.texi | ./Tidytxt >filter.texinfo
 	      /bin/rm -rf exim_filtering.texi
@@ -202,6 +203,7 @@ spec.txt: spec-txt.html Tidytxt
 
 spec.info:    spec-info.xml
 	      $(DOCBOOK2TEXI) spec-info.xml
+	      if test -e spec.texi ; then mv spec.texi the_exim_mta.texi ; fi
 	      ./TidyInfo <the_exim_mta.texi >spec.texinfo
 	      /bin/rm -rf the_exim_mta.texi
 	      makeinfo -o spec.info --no-split spec.texinfo
--- a/doc-docbook/filter.xfpt
+++ b/doc-docbook/filter.xfpt
@@ -67,6 +67,7 @@
 <bookinfo>
 <title>Exim's interfaces to mail filtering</title>
 <titleabbrev>Exim filtering</titleabbrev>
+<titleabbrev role="texinfo-file">filter</titleabbrev>
 <date>
 .fulldate
 </date>
--- a/doc-docbook/spec.xfpt
+++ b/doc-docbook/spec.xfpt
@@ -173,6 +173,7 @@
 <bookinfo>
 <title>Specification of the Exim Mail Transfer Agent</title>
 <titleabbrev>The Exim MTA</titleabbrev>
+<titleabbrev role="texinfo-file">spec</titleabbrev>
 <date>
 .fulldate
 </date>
